src/containers/LearnerDashboardHeader/index.test.jsx

```diff
@@ -39,4 +39,9 @@ describe('LearnerDashboardHeader', () => {
     const wrapper = shallow(<LearnerDashboardHeader />);
     expect(wrapper.instance.findByType(Header)[0].props.secondaryMenuItems.length).toBe(1);
   });
+  test('should display Programs link if the service is configured in the backend', () => {
```

Comment:
Looks like we need to reword the test title, here.

```suggestion
  test('should display Programs link if it's enabled by configuration', () => {
```